boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -47,21 +47,6 @@
 			};
 		};
 
-		etr_buf_ram0x_region: memory@2f0be000 {
```

Comment:
I don't think we need to take this out of the default memory map. It already contains RAM regions which are sometimes unused, but at least we get to keep a uniform base layout for multiple samples/tests.